src/sentry/uptime/subscriptions/regions.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
+
+from sentry.conf.types.uptime import UptimeRegionConfig
+
+region_lookup = None
+
+
+def get_active_region_configs() -> list[UptimeRegionConfig]:
+    return [v for v in settings.UPTIME_REGIONS if v.enabled]
+
+
+def get_region_config(region_slug: str) -> UptimeRegionConfig | None:
+    region = get_region_lookup().get(region_slug)
+    if region is None:
+        # XXX: Temporary hack to guarantee we get a config
+        region = get_active_region_configs()[0]
```

Comment:
Why have the hack in place right now, looks like this can return `None` right?

Reply:
I was pulling out code from other prs - I want to switch existing code to get the region configs from here, so for subscriptions with no region I want them to get the default.